pages/dkp/kommander/2.1/security/policy-controls/gatekeeper/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: layout.pug
+navigationTitle: Enforce policies using Gatekeeper
+title: Enforce policies using Gatekeeper
+menuWeight: 24
+excerpt: Learn how to enforce policies using Gatekeeper
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD030 MD018-->
+
+[Gatekeeper][gatekeeper] is the policy controller for Kubernetes, allowing organizations to enforce configurable policies using the [Open Policy Agent][opa], a policy engine for Cloud Native environments hosted by CNCF as an incubation-level project.
+
+This tutorial demonstrates how to use Gatekeeper to enforce policies by rejecting non-compliant resources. Specifically, this tutorial will demonstrate two constraints as a way to show how to use Gatekeeper as an alternative to [Pod Security Policies][psp]:
+
+- [Prevent the running of privileged pods](#prevent-privileged-pods)
+- [Prevent mounting host path volumes](#prevent-host-path-volumes)
+
+## Before you begin
+Before starting this tutorial, you should verify the following:
+
+- You must have access to a Linux, macOS, or Windows computer with a supported operating system version.
+
+- You must have a properly deployed and running cluster. For information about deploying Kubernetes with default settings on different types of infrastructures, see the [Choose Infrastructure][choose-infrastructure].
+
+- Ensure Gatekeeper is [enabled](../../../install/http-proxy/#enable-gatekeeper)
```

Comment:
since gatekeeper is enabled by default, I wonder if we can drop the link. Also, that page describes how to enable mutating webhooks which we don't really need for this doc (which describes validating webhooks i think)

Reply:
I think the link provides more info than we need. I dropped the link and changed the wording.